lib/pundit/matchers/base_matcher.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'utils/policy_info'
+
+module Pundit
+  module Matchers
+    # This is the base class for all matchers in the Pundit Matchers library.
+    class BaseMatcher
+      # Error message when an ambiguous matcher is used.
+      AMBIGUOUS_MATCHER_ERROR = <<~MSG
+        `expect().%<matcher>s` is not supported since it creates ambiguity.
+        Instead use `expect().%<suggested_matcher>s`'
```

Comment:
I'll remove the suggestion because no one can understand what `not_to forbid_all_actions` means